### PR TITLE
Ignore changes in git submodules

### PR DESCRIPTION
**Pull request by imbsky**

### CODE REVIEW

**Comment by imbsky** [comment]
`dune build` causes changes because Dune rewrites inside the vendor directory. In this PR, ignore that by modify the `.gitmodules` file.

File: .gitmodules [hunk 1]
```diff
@@ -1,4 +1,5 @@
 [submodule "vendor/merlin"]
-	path = vendor/merlin
-	url = https://github.com/rgrinberg/merlin
-	branch = remove-lsp
```
[the remark this answers is not in the document]
Comment (imbsky) [reply .gitmodules]: We don't need indentation of the git config, I can add it if you want, but some editors automatically delete it.

[the remark this answers is not in the document]
Comment (rgrinberg) [reply .gitmodules]: Yeah, let's leave it in. I like the style that git chooses by default here.

[the remark this answers is not in the document]
Comment (imbsky) [reply .gitmodules]: Updated!

**Comment by imbsky** [comment]
@rgrinberg Ready to merge 👍

**Pull request merged**